core/src/main/scala/org/apache/spark/rdd/HadoopRDD.scala

```diff
@@ -270,7 +270,8 @@ class HadoopRDD[K, V](
     val iter = new NextIterator[(K, V)] {
 
       private val split = theSplit.asInstanceOf[HadoopPartition]
-      logInfo(log"Input split: ${MDC(INPUT_SPLIT, split.inputSplit)}")
+      logInfo(log"Task (TID ${MDC(TASK_ID, context.taskAttemptId())}) input split: " +
```

Comment:
Use `TASK_ATTEMPT_ID`?

Reply:
`context.taskAttemptId()` is task id, this was clarified in https://github.com/apache/spark/pull/45834#discussion_r1635746525